save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/execution/TestExecutionFilters.kt

```diff
@@ -0,0 +1,24 @@
+package com.saveourtool.save.execution
+
+import com.saveourtool.save.domain.TestResultStatus
+
+/**
+ * Aff filters in one property
+ * @property status to filter by [status]
+ * @property fileName to use filter by [fileName]
+ * @property testSuite to use filter by [testSuite]
+ * @property tag to use filter by [tag]
+ */
+data class TestExecutionFilters(
+    var status: TestResultStatus?,
+
+    var fileName: String?,
+
+    var testSuite: String?,
+
+    var tag: String?,
```

Comment:
Why do you need it to be `var`? Make it `val` pls

Reply:
but I change the values of this data in the process

Reply:
> but I change the values of this data in the process

You shouldn't be changing fields of property in React state, because it doesn't trigger update

Reply:
Actually, `setState(S.() ->)` from  `kotlin-wrappers` calls `clone(obj)` under the hood which leads to correct update of state object. The other reason to use immutable class is to avoid accidental mutations and to make it cleaner during data exchange